Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -794,23 +756,23 @@ - (void) receivedHTTPResponse: (CFHTTPMessageRef)httpResponse {
     [self callC4Socket:^(C4Socket *socket) {
         c4socket_gotHTTPResponse(socket, (int)httpStatus, {headersFleece.buf, headersFleece.size});
     }];
-
+    
     if (httpStatus != 101) {
         // Unexpected HTTP status:
         C4WebSocketCloseCode closeCode = kWebSocketClosePolicyError;
         if (httpStatus >= 300 && httpStatus < 1000)
             closeCode = (C4WebSocketCloseCode)httpStatus;
         [self closeWithCode: closeCode reason: _logic.httpStatusMessage];
-
+        
```

Comment:
No need for an extra empty line here.

Reply:
it is not a new empty line, its auto formatting tab space. But right, there is no need for line as well.